meetups/20180502.org

```diff
@@ -0,0 +1,130 @@
+
+* Tech Check
+*** Wi-Fi
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+
+- SSID "Triplebyte Guest"
+- Password "welcome01"
+* Introduction
+*** Call for Engagement
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+- More venues are always good
+- If you're an "event organizer" you can create any meeting you like
+  - Emacs and Clojure?
+  - Meetings in South/East/North Bay?
+  - More convenient times?
+*** Github Repo
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+https://github.com/Emacs-SF/meetup-notes
+*** Future Topics
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+- Specific guest speakers?
+  - I've run out for now
+  - Maybe one of us has a full-length talk in them?
+- Programming Languages: Python, Clojure, C++
+- Elisp programming?
+- Minor modes?
+- Please LMK or post ideas in group
+*** org-reveal
+#+NAME: Setup
+#+BEGIN_SRC emacs-lisp
+(use-package ox-reveal
+  :config
+  (setq org-reveal-root "https://cdn.jsdelivr.net/npm/reveal.js@3.6.0")
+)
+#+END_SRC
+
+Export with C-c C-e R R, then load file in browser
+* Sponsor
+*** Many thanks to
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+[[file:./triplebyte_logo.png]]
+* Presenter
+*** Harry Schwartz
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+
+** Introduction
+- Harry does a lot of arithmetic at work
+- Today we will all stop opening terminals to calculate numbers
+** Calc
+*** Overview
+- An RPN-based calculator
+- M-x calc
+*** Basic usage
+- type numbers to add them onto the stack
+- type operators (+-*/^) to perform the operation and replace the top of the stack with it
+- need to quote minus sign to enter negative. Use single quote
+- capital U to undo
+- TAB rotates top 2 numbers on stack
+- M-TAB rotates the top 3
+- Harry notes you can do arbitrary manipulations with those two alone
+- ?hh for help
+*** Vectors
+- Used for numeric vectors but also things like sets
+- can do inner and outer products, LU decomposition
+
+*** Algebra
+- The same single quote that introduced negative numbers introduces algebra
+- macsyma format
+- many commands introduced by "a" to do manipulations
+  - expansion, factoring, substitutions (with parens introduced where needed)
+
+*** Integration with External Tools
+- gnuplot
+  - [-10..10] inclusive range
+  - x^2
+  - "gf" will now graph x^2 over the range
+  - you can also do range intersections (attempted live!)
+  - "gn" chooses number of points to plot
+  - "gF" does a 3D plot!
+
+*** Other things that came up
+- there is a macro called defmath you can define your own calc functions with
+- "screenkey" was the tool used to display keys being pressed
+- Harry couldn't find the info files for calc; it turns out they are in a separate package ("emacs25-common-non-dfsg") on Debian (and thus Ubuntu) for license reasons.
+
+** Post-Presentation Discussion
+- We veered off pretty far, into math and category theory
+- Session Idea: how to write an org-mode exporter
+
```

Comment:
My other notes (you could add here?):
1. online calc info: 
    > https://www.gnu.org/software/emacs/manual/html_mono/calc.html
2. on Debian, emacs info for calc is missing unless you install it, e.g.:
    $ sudo apt-get install emacs25-common-non-dfsg
3. there was a question about what tool was used to echo keystrokes.  It was screenkey, here's a link: 
    >  https://github.com/wavexx/screenkey
Category theory
Software foundations book (I found the following link, is it correct?):
> https://softwarefoundations.cis.upenn.edu/

and I've been enjoying the book you mentioned in the meetups comments "From Mathematics to Generic Programming" (which I found in my safaribooksonline.com account)
Here's a link to the book on Amazon:
> https://smile.amazon.com/Mathematics-Generic-Programming-Alexander-Stepanov/dp/0321942043/

Reply:
I got screenkey and the info package in there already but not the others :) Thanks!